CHANGELOG.md

```diff
@@ -11,14 +11,22 @@ and this project adheres to
 ### Added
 - schema: Add support of optional list of GPU on _NodeType_ (from
   @btravouillon).
+- core: Add `bits` defined type.
 - docs:
   - Mention web extra package installation from PyPI in quickstart guide.
   - Mention new optional list of _NodeTypeGpu_ on _NodeType_ in OpenAPI
     specification and structure reference documentation (from @btravouillon).
   - Add nodetype with GPU in examples databases.
+  - Mention new `~bits` defined type.
 
 ### Changed
-- docs: Update supported Linux distributions in quickstart guide.
+- schema: Use `~bits` defined type instead of `~bytes` for _NodeTypeNetif_,
+  _StorageEquipmentTypeNetif_ and _NetworkEquipmentTypeNetif_ bandwidth
+  properties (#21).
+- docs:
+  - Update supported Linux distributions in quickstart guide.
```

Comment:
This is not in this PR. Is this a catch up of a previous change?

Reply:
This line was above, its order has simply changed in this PR.